src/aof.c

```diff
@@ -1610,7 +1608,7 @@ int rewriteAppendOnlyFile(char *filename) {
         /* Update COW info */
         long long now = mstime();
         if (now - cow_updated_time >= 1000) {
-            sendChildCOWInfo(CHILD_INFO_TYPE_CURRENT_COW_SIZE, "AOF rewrite");
+            sendChildInfo(CHILD_INFO_TYPE_CURRENT_INFO, 0, "AOF rewrite");
```

Comment:
I think we wanna send the total key count here. 
(and remove the `if` that ignores values of 0)

Reply:
We send here the AOF diff, we don't have here the AOF diff keys count data only the bytes, neither we have the total key count which is saved by the parent only.

Reply:
we do have the key count, we just finished iterating over them.
we could in theory have `rdbSaveRio` and `rewriteAppendOnlyFileRio` return their `key_count` variable to the caller.
but there's also no real reason why we can't just call `dbTotalServerKeyCount` here (preferably once outside the loop)

We can argue about what's cleaner (this or the `if` in the parent), but note that doing that "solves" a "bug".
since the code only sends the key count once in a second, if this loop aof diff loop runs for a long time, `current_save_keys_processed` will be showing the last value for a long time, which isn't reflecting the reality.

Reply:
Oh, I thought you meant the AOF diff's keys count.
Done.